vespalib/src/vespa/vespalib/util/growstrategy.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <cstdint>
+
+namespace vespalib {
+
+class GrowStrategy {
+private:
+    uint32_t _initialCapacity;
```

Comment:
Consider changing all use of uint32_t to size_t to match what is used in rcuvector.

Reply:
Done